formatMessage.go

```diff
@@ -83,6 +83,7 @@ func FormatMessage(ctx context.Context, emailReq EmailRequest, payload map[strin
 	payload["DEToolsLink"] = deSettings.base + deSettings.tools
 	payload["DECollectionsLink"] = deSettings.base + deSettings.collections
 	payload["DEAppsLink"] = deSettings.base + deSettings.apps
+	payload["DEAnalysesLink"] = deSettings.base + deSettings.analyses
```

Comment:
Looks like this isn't being used anywhere?

Reply:
Oh, right. I was going to add this as a fallback in case we don't have an analysis ID. I'll add that to the template now.